prompt2model/input_generator/prompt_template.py

```diff
@@ -23,79 +23,81 @@
 
 {example_4}
 --------------------------------------------------------------------------------------------
-Here are some [high-quality inputs] for the [new instruction]. These inputs can provide you with very strict format requirements. You should pay extreme attention to them!!!
+Here is the new [instruction] for a task.
 
-[high-quality inputs]:
+new [instruction]={instruction}
+--------------------------------------------------------------------------------------------
+Here are some high-quality [input] for the [instruction]. These [inputs] can provide you with very strict format requirements. You should pay extreme attention to them!!!
+
+Some high-quality [inputs]:
```

Comment:
How about also changing into [input] here?

Reply:
Done! You can definitely suggest more changes.